Apsitvarkom.Models/Public/TidyingEventResponse.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Apsitvarkom.Models.Public;
+
+public class TidyingEventResponse
+{
+    public Guid Id { get; init; }
+
+    public Guid PollutedLocationId { get; set; }
+
+    public DateTime StartTime { get; set; }
+
+    public string? Notes { get; set; }
+}
```

Comment:
Didn't think there was any need for a validator here, hence it's empty.

Reply:
Response validators are used to document swagger (show schema), for example that the `Id` is not nullable

Reply:
Nevermind, nullable is determined automatically, these validators to document swagger with **business rules**, but this model doesn't really have any right now